ipyvizzu.py

```diff
@@ -168,6 +198,18 @@ def build(self):
         return {"data": self}
 
 
+class Filter(Animation):
+    def __init__(self, data: typing.Optional[str]):
+        self._data = (
+            RawJavaScript(f"record => {{ return ({data}) }}")
+            if data is not None
+            else RawJavaScript(data)
```

Comment:
Could we skip `RawJavaScript` in case of None?

```python
self._data = RawJavaScript(f"record => {{ return ({data}) }}") if data is not None else data
```

So `RawJavaScriptEncoder` should not have to handle `null`

Reply:
Thanks, I simplified it.